tests/integration/test_s3.py

```diff
@@ -106,7 +106,7 @@ def test_s3_upload_fileobj_with_large_file_notification(self):
         self._create_test_notification_bucket(queue_attributes)
 
         # has to be larger than 64MB to be broken up into a multipart upload
-        file_size = 75000000
+        file_size = 400 * 1024 * 1024
```

Comment:
Thanks for making the `S3_MAX_FILE_SIZE_MB` change. Can we leave this at `75000000` - this is sufficient to test the multipart logic in this test, and I'd like to avoid adding unnecessary strain on the tests we're running in CI.